src/util/rocksdb/mod.rs

```diff
@@ -171,17 +175,13 @@ fn check_and_open(
 
     // Create needed column families not existed yet.
     for cf in cfs_diff(&needed, &existed) {
-        db.create_cf(
-            cf,
-            cfs_opts
-                .iter()
-                .find(|x| x.cf == cf)
-                .unwrap()
-                .options
-                .clone(),
-        )?;
+        if let Some(opts) = cfs_opts.iter().find(|x| x.cf == cf) {
+            let opt = opts.options.clone();
+            db.create_cf((cf, opt))?;
+        } else {
+            return Err(format!("can't find cf: {}", cf));
```

Comment:
It won't happen.

Reply:
I just think we should avoid `unwrap`. How about `panic` with a message here?

Reply:
There is no big difference between `unwrap` and `panic` since it's always unreachable. I prefer `unwrap` for its simplicity (and "covered").

Reply:
got it.